source/common/stats/grpc_metrics_service_impl.h

```diff
@@ -135,6 +135,26 @@ class MetricsServiceSink : public Sink {
     gauage_metric->set_value(value);
   }
 
+  void flushHistogram(const Histogram& histogram) override {
```

Comment:
as discussed, i am just pushing the summary quantiles not rich histogram in the first iteration. If some one is interested, I am open to publishing rich histogram data as well.